www/app/blog/2023-12-18-transaction-isolation-postgres.mdx

```diff
@@ -599,40 +599,37 @@ What are serialization errors? They look something like this: `ERROR: could not
   <tbody>
     <tr>
       <td>
-        <pre>BEGIN;</pre>
-      </td>
-      <td>
-        <pre>BEGIN;</pre>
+        {/* prettier-ignore */}
+        <pre>BEGIN TRANSACTION ISOLATION LEVEL SERIALIZABLE;
```

Comment:
is it possible for this to be

\`\`\`sql
BEGIN TRANSACTION ISOLATION LEVEL SERIALIZABLE;
select * from accounts
\`\`\`

Reply:
Let me try that. I believe I moved to <pre> because I had an issue with ```. But now I don't remember what it was...

Reply:
I tried and this does look better. It goes a bit wide, but still better to have the highlighting and all. Thank you! 

I only fixed this example here, will do a separate PR with the rest of the code formatting.